### DISCUSSION TITLE
How to run Kafka UI in network_mode host?

### DISCUSSION THREAD

**Discussion by pejeio** [provectus/kafka-ui#3925] [Q&A] [answered]
When running Kafka UI in [network_mode](https://docs.docker.com/compose/compose-file/compose-file-v3/#network_mode) `host`, the UI is not available on http://localhost:8080

No other processes are active on port 8080 on the host machine.

**Comment by Haarolean** [comment, marked as answer]
It works for me on linux. If you're running it on mac, be aware of https://github.com/docker/compose/issues/3800 and https://github.com/docker/for-mac/issues/1031
